stl/inc/format

```diff
@@ -541,10 +542,20 @@ constexpr const _CharT* _Parse_format_specs(const _CharT* _Begin, const _CharT*
 
     if (*_Begin == '.') {
         _Begin = _Parse_precision(_Begin, _End, _Callbacks);
+        if (_Begin == _End) {
+            return _Begin;
+        }
+    }
+
+    if (*_Begin == 'L') {
+        _Callbacks._On_localized();
+        if (++_Begin == _End) {
+            return _Begin;
+        }
     }
 
     // If there's anything remaining we assume it's a type.
-    if (_Begin != _End && *_Begin != '}') {

```

Comment:
is this just removed for being redundant?

Reply:
It was needed before, but I moved the check into testing against '.' and 'L', so yeah now it's redundant.